// [[Reason: TSC adds type reference directives.]] ////

//// [tests/cases/compiler/moduleAugmentationsImports2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -18,9 +18,8 @@
     }
 }
 
 //// [e.d.ts]
-/// <reference path="c.d.ts" />
 import { Cls } from "C";
 declare module "./a" {
     interface A {
         getCls(): Cls;
@@ -29,57 +28,4 @@
 
 //// [main.d.ts]
 import "d";
 import "e";
-
-/// [Errors] ////
-
-e.ts(2,1): error TS9024: Declaration emit for this expression requires adding a type reference directive to '.c' with --isolatedDeclarations.
-
-
-==== a.ts (0 errors) ====
-    export class A {}
-    
-==== b.ts (0 errors) ====
-    export class B {x: number;}
-    
-==== c.d.ts (0 errors) ====
-    declare module "C" {
-        class Cls {y: string; }
-    }
-    
-==== d.ts (0 errors) ====
-    /// <reference path="c.d.ts"/>
-    
-    import {A} from "./a";
-    import {B} from "./b";
-    
-    A.prototype.getB = function () { return undefined; }
-    
-    declare module "./a" {
-        interface A {
-            getB(): B;
-        }
-    }
-    
-==== e.ts (1 errors) ====
-    import {A} from "./a";
-    import {Cls} from "C";
-    ~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS9024: Declaration emit for this expression requires adding a type reference directive to '.c' with --isolatedDeclarations.
-    
-    A.prototype.getCls = function () { return undefined; }
-    
-    declare module "./a" {
-        interface A {
-            getCls(): Cls;
-        }
-    }
-    
-==== main.ts (0 errors) ====
-    import {A} from "./a";
-    import "d";
-    import "e";
-    
-    let a: A;
-    let b = a.getB().x.toFixed();
-    let c = a.getCls().y.toLowerCase();
\ No newline at end of file
